tests/Feature/Controllers/Email/EmailIndexControllerTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Tests\Feature\Controllers\Email;
+
+use App\Models\Email;
+use App\Models\EmailTemplate;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class EmailIndexControllerTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /** @test */
+    public function it_can_show_emails_index()
```

Comment:
I think is a functional test you should check only get the result 

 $response = $this->get('/email');
 $response->assertStatus(200);

The rest of the code should be on another kind of test in an Unit test

Reply:
I have added a search_emails test to a separate logic